volume/drivers/pwx/connection_test.go

```diff
@@ -144,7 +144,7 @@ func TestPortworx_buildClientsEndpoints_Error_WhenServiceDoesNotExist(t *testing
 	cleaner := setEnvs(t, pxEnableTLSEnv, "true", pxServiceNameEnv, "portworx-service-not-found", pxNamespaceNameEnv, "kube-system")
 	defer cleaner()
 
-	paramsBuilder, err := pwx.NewConnectionParamsBuilder(ops, pwx.NewConnectionParamsBuilderDefaultConfig())
+	paramsBuilder, err := pwx.NewConnectionParamsBuilder(ops, pwx.NewConnectionParamsBuilderDefaultConfig(), nil)
```

Comment:
As part of the Config, there would be no need to pass `nil` as a new parameter. Which makes the code easier to read.

Reply:
fixed